packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -100,6 +110,23 @@ class ChromeDriverInstaller {
     return int.parse(versionNo);
   }
 
+  /// Find Google Chrome App on Mac.
+  Future<String> findChromeExecutableOnMac() async {
```

Comment:
I think this can be shortened to:

```dart
Directory('/Applications').listSync().whereType<Directory>()
  .firstWhere(
    (d) => path.basename(d.path).endsWith('Chrome.app'),
    orElse: () => throw Exception('Failed to locate system Chrome'),
  );
```

Reply:
this is great thanks! worked on my mac.

This assumes it always directly under /Applications. I will use this approach. If we see this is not the case in every MacOS, then we can change it back to find and search everywhere! (which is a way slower)

Reply:
AFAIK apps are always directly under `/Applications`